### DISCUSSION TITLE
Any ideas on how to add this into a Function App in Azure?

### DISCUSSION THREAD

**Discussion by huezodaniel** [G-Research/ParquetSharp#416] [Q&A]
Hi all,
i have implemented a code to read from blob storage and simply getting a max date from it. Locally is working but unfortunately when i deploy to my function app it keeps saying that it cannot find the ParquetSharpNative.dll. 
Any ideas?

**Comment by adamreeve** [comment]
I don't have any experience with Azure Functions, but there is some discussion here that might be helpful: https://github.com/Azure/Azure-Functions/issues/622
